integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFormatsIT.java

```diff
@@ -120,6 +122,31 @@ public void testNumericFormats() {
         rows("1970-01-02 03:55:00", "1970-01-01 00:01:40.5"));
   }
 
+  @Test
+  @SneakyThrows
+  public void testDateNanosWithFormats() {
+    String query = String.format("SELECT hour_minute_second_OR_t_time"
+        + " FROM %s", TEST_INDEX_DATE_FORMATS);
+    JSONObject result = executeQuery(query);
+    verifySchema(result,
+        schema("hour_minute_second_OR_t_time", null, "time"));
```

Comment:
can you change this to `hour_minute_second_millis` instead?  The point of using the `date_nano` type is to include fractions of a second.

Reply:
I use `t_time` here, it includes up to 9 digits of fsp.

Reply:
but the verifyDataRows doesn't include any units of precision?  Maybe include data that has fractions of a second?